File: adk/tests/fixtures/unit_profiles/python-pytest/project/calculator.py
"""Código mínimo usado na demonstração do perfil Python/pytest."""


def add(left: int, right: int) -> int:
    return left + right


def divide(dividend: float, divisor: float) -> float:
    if divisor == 0:
        raise ValueError("divisor must not be zero")
    return dividend / divisor

File: adk/tests/fixtures/unit_profiles/node-vitest/project/src/calculator.ts
export function add(left: number, right: number): number {
  return left + right;
}

export function divide(dividend: number, divisor: number): number {
  if (divisor === 0) {
    throw new RangeError("divisor must not be zero");
  }
  return dividend / divisor;
}
